### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by mkallies** [adamwalzer/vscode-scss-lint#2]
Currently trying out VSCode and I use scss-lint for my project.  I've been using scss-lint just fine in Atom but for some reason it's not working in VSCode with you're extension.  I currently have scss_lint 0.49.0 installed with your extension version 0.0.6

**Comment by WolfieZero** [comment]
I'm having the same issue as well.

.scss-lint.yml is in the root of my project

*Versions*
ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-darwin16]
scss-lint 0.49.0
vs code 1.10.1

**Comment by BProg** [comment]
I am having the same problem on windows.

**Comment by maciejmyslinski** [comment]
Same for me. mac os.

**Comment by Dorza** [comment]
Same for me, I usually run atom (works fine) but need to test on VSC as some of our team use this also. So I downloaded VSC, install the scss-lint plugin save a file that i know causes errors in atom and I get nothing. As I say this works fine in Atom, I have my yml file at the root of my project and have the correct gem installed.

Windows 10

**Comment by gavdevs** [comment]
any body figure out how to fix this?

**Comment by gavdevs** [comment]
or alternatives?

**Comment by yoodame** [comment]
You have to put the `.scss-lint.yml` file in your root directory for it to work. I submitted a [pull request](https://github.com/adamwalzer/vscode-scss-lint/pull/3) that lets you put the config in subdirectories. Just waiting on @adamwalzer  to review and accept.

https://github.com/adamwalzer/vscode-scss-lint/pull/3

**Comment by adamwalzer** [comment]
@yoodame's PR solves this issue. I'm going to close it. Thanks!

**Comment by donni106** [comment]
It is still not working for me.

@WolfieZero was mentioning, that
> I'm having the same issue as well.
> .scss-lint.yml is in the root of my project

@yoodame said for fixing
> You have to put the .scss-lint.yml file in your root directory for it to work.
> I submitted a pull request that lets you put the config in subdirectories.

So it seems not to be the fix for the problems some of us have with .scss-lint in root?!

My scss-lint is working from terminal, but VS Code always shows <img width="131" alt="bildschirmfoto 2017-05-23 um 12 08 18" src="https://cloud.githubusercontent.com/assets/1942953/26349603/8b059542-3fb0-11e7-96df-9ada4711402a.png">.

**Comment by adamwalzer** [comment]
I'm not sure what your issue is, but if the supplied solution is solving this issue for other people perhaps we should open another issue for your troubles. Please explain it well so we can debug it. I will be traveling the next couple weeks so I won't be able to publish any updates anytime soon. I did however add support for css if that is your issue. The README explains how to add css support to your project.

**Comment by donni106** [comment]
It is still the same issue like i.e. @Dorza was commenting 11 days ago:

> Same for me, I usually run atom (works fine) but need to test on VSC as some of our team use this also. So I downloaded VSC, install the scss-lint plugin save a file that i know causes errors in atom and I get nothing. As I say this works fine in Atom, I have my yml file at the root of my project and have the correct gem installed.

For me it is not Atom but Sublime, where scss linting works fine and in VS Code not.
@yoodame fix is targeting config files in subdirectories. It is not the fix for the default setup with .scss-lint.yml in the root path of a project, or am I wrong?

**Comment by adamwalzer** [comment]
Hmm. Alright. What version of the extension do you have? And could you show me what you run in your terminal input and output? Thanks.

**Comment by donni106** [comment]
In terminal I can run scss-lint properly:

> ➜ scss-lint app/assets/stylesheets/blog/_blog.scss

Output:
> app/assets/stylesheets/blog/_blog.scss:1:1 [W] SelectorFormat: Selector `blog_page_splash` should be written in lowercase with hyphens
app/assets/stylesheets/blog/_blog.scss:2:3 [W] PropertySortOrder: Properties should be ordered align-items, background-color, background-image, background-position, background-size, display, height, paddin
g, text-align, width
app/assets/stylesheets/blog/_blog.scss:4:31 [W] StringQuotes: Prefer single quoted strings
...

When opening and working in the file: <img width="133" alt="bildschirmfoto 2017-05-24 um 11 43 32" src="https://cloud.githubusercontent.com/assets/1942953/26397335/3f033ac0-4076-11e7-9271-6116069ef8e8.png">


scss_lint v 0.49.0
scss-lint VS Code plugin v 0.0.13
macOS Sierra 10.12.5

**Comment by yoodame** [comment]
@donni106 I have the same versions as well. The extension didn't work for me this morning. I tried reloading vscode but reinstalling the extension worked. Maybe give that a shot if you haven't already.

It is strange that it was working for me the day before.

**Comment by donni106** [comment]
Thanks @yoodame! I tried to uninstall/install several times with reopening VS Code without success.
**BUT!** I checked my different ruby versions installed per rbenv and found out, that scss_lint was not installed in the system ruby.
Installing it there made it work for VS Code!

Now I can see the scss-lint errors directly in code. Ironically the counter in the bottom bar stays at 0.

**Comment by adamwalzer** [comment]
The counter is displaying errors. Those were all warnings, not errors. Would we like to display warnings as well? I could adjust how that works.

**Comment by donni106** [comment]
Ah good to know, thanks. I dive deeper and deeper.
I opened a new issue for showing warnings.

**Comment by adamwalzer** [comment]
Ok awesome. I'm going to close this issue then.
